### PR TITLE
use new modules syntax

### PR DESCRIPTION
**Pull request by Dhaulagiri**
updates code to use new modules api from [rfc 176](https://github.com/emberjs/rfcs/pull/176)

### CODE REVIEW

File: addon/components/markdown-to-html.js [hunk 1]
```diff
@@ -1,19 +1,20 @@
...
+import { get, computed } from '@ember/object';
+import { merge, assign } from '@ember/polyfills';
+import { getOwner } from '@ember/application';
 import showdown from 'showdown';
 import layout from '../templates/components/markdown-to-html';
 
-const { computed, get, merge, getOwner } = Ember;
 const CONFIG_LOOKUP = 'config:environment';
 
-let assign = Ember.assign;
```
Comment (Dhaulagiri) [inline addon/components/markdown-to-html.js]: not super familiar with what is going on here, but this change is making all the tests blow up.  any ideas?

Comment (jasonmit) [reply addon/components/markdown-to-html.js]: I'll fix this, merging first so that I don't have to fork your fork

Comment (Dhaulagiri) [reply addon/components/markdown-to-html.js]: Cool, I’ll check that out

Comment (jasonmit) [reply addon/components/markdown-to-html.js]: All fixed - looks like assign/merge were never used so was an easy fix (removed them).  I'll publish a new minor version tonight.

Comment (Dhaulagiri) [reply addon/components/markdown-to-html.js]: Nice, I thought that might be the case but couldn’t figure out based on the blame why it was added in the first place.

**Comment by jasonmit** [comment]
Released as 4.1.0, thanks!

**Pull request merged**